src/views/HelpAndKnowledge/Articles/_components/HelpCardGroup/index.tsx

```diff
@@ -2,13 +2,13 @@ import React, { useState } from 'react';
 import ReactPaginate from 'react-paginate';
 import { CardGroup } from '@trussworks/react-uswds';
 
-import ArticleCard from 'components/ArticleCard';
+import ArticleCard from 'views/HelpAndKnowledge/Articles/_components/ArticleCard';
 
 import helpAndKnowledgeArticles from '../..';
 
 type HelpCardGroupType = {
   className?: string;
-  filter?: 'itImplementation' | 'gettingStarted';
+  filter?: 'it-implementation' | 'getting-started' | string | null;
```

Comment:
Doesn't adding the `string` type kinda defeats the purpose of explicit spelling out the types `'it-implementation` and `getting-started`?

Reply:
That param is coming direectly from a URL optional param, so it has to be `string` | `null` to comply.  Not really sure atm how to enforce this